experiments.sh

```diff
@@ -0,0 +1,25 @@
+export HF_DATASETS_CACHE="/gpfswork/rech/tts/unm25jp/datasets"
```

Comment:
Why do we need this? Also, similar to above, I think we should remove hardcoded non-shared paths.